node/narwhal/Cargo.toml

```diff
@@ -107,3 +107,7 @@ features = [ "fs", "trace" ]
 [dev-dependencies.tracing-subscriber]
 version = "0.3"
 features = [ "env-filter" ]
+
+[dev-dependencies]
+test-strategy = { git = "https://github.com/frozenlib/test-strategy/", branch = "master"}
```

Comment:
Will this impact our ability to `cargo publish` this crate?

Reply:
There are workarounds, see: https://github.com/rust-lang/cargo/issues/4242 but it's not ideal.

Reply:
I tested this in another repo and it seems to work fine, I think we can green-light this PR.

Reply:
@niklaslong I believe this behavior is also properly documented here: https://github.com/rust-lang/cargo/pull/7333. This is default behavior since 1.40 for dev-dependencies without a specified `version`.